ibm_watson/websocket/recognize_listener.py

```diff
@@ -196,16 +196,20 @@ def on_data(self, ws, message, message_type, fin):
         elif 'results' in json_object or 'speaker_labels' in json_object:
             hypothesis = ''
             if 'results' in json_object:
-                hypothesis = json_object['results'][0]['alternatives'][0][
-                    'transcript']
-                b_final = (json_object['results'][0]['final'] is True)
-                transcripts = self.extract_transcripts(
-                    json_object['results'][0]['alternatives'])
+                if len(json_object['results']) != 0:
+                    # Handling for noisy/silent utterances
+                    print(json_object['results'])
```

Comment:
do we need the `print` statement here?

Reply:
The print statement is not needed, it can be removed or replaced.

Reply:
Sorry forgot about this PR, if we get rid of the print statement we should be good to go.